contracts/farm/PriceGetterP125.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity 0.6.12;
```

Comment:
make it range?

Reply:
since it  new probably ^0.8.0

Reply:
OracleLibrary.sol does not support solidity versions >= 0.8.0 so was limited. I chose 0.6.12 but can change it to ^0.7.0 to be on a newer version.